about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,100;1,300;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="about.css">
+    <title>Document</title>
+</head>
+<body>
+    <section id="main-nav-menu">
+        <div class="sub-nav-bar">
+            <ul id="list-sub-nav-bar">
+                <li id="facebook"><img class="nav-inside" src="images+/facebook.png" alt=""></li>
+                <li id="twitter"><img class="nav-inside" src="images+/gorjeo.png" alt=""></li>
+                <li id="login"><a class="nav-inside" href="#"></a>Login</li>
+                <li id="mypage"><a class="nav-inside" href="#"></a>MyPage</li>
+                <li id="english"><a class="nav-inside" href="#"></a>English</li>
+                <li id="menu"><a class="menuu" href="#"><img src="images+/Icon-Menu.jpg" alt="main-menu"></a></li>
+                <a href="index.html" ><img id="logo" src="./images+/logo.jpg" alt="Logo"></a>
+            </ul>
+        </div>
+        <header class="menu-bar">
+            <nav class="content-nav">
+              <div>
+                <a href="index.html" ><img id="logoo" src="./images+/logo.jpg" alt="Logo"></a>
+              </div>
+             <div>
+                <ul id="menu-links">
+                    <li><a class="menu-link" href="index.html">Home</a></li>
+                    <li><a class="menu-link" href="about.html">About</a></li>
+                    <li><a class="menu-link" id="contact" href="#">Contact</a></li>
+                </ul>
+             </div>
+            </nav>
+          </header>
+        
+        <div id="main-section">
+            <div>
+                <h2 id="sub-title">Hello!!! Introducing to the world</h2>
+                <h1 id="title">Home Decoration Page 2022</h1>
+                <h3 id="sub-title-2">In this section about us, we will tell you how our launch to the market was and how we were able to locate ourselves</h3>
+                <br>
+            </div>
```

Comment:
- [ ] Kindly make sure to adjust the title styling in order to meet the provided design.

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/191822420-4750b6ae-dfad-4167-b3f3-7700c0d7b8a0.png)

  - ### Behance design

   ![image](https://user-images.githubusercontent.com/87197412/191822307-1b99990b-b803-45fc-96d1-3b544bccf434.png)